rad/monadic/items.rad

```diff
@@ -199,30 +248,30 @@
     (def items-dict (dict-from-seq filtered-items))
     (if (empty-seq? filtered-items)
       (no-filtered-items)
-      (if (lookup :fancy options)
-        (match (fzf-select-with-preview!
-                 (map first filtered-items)
-                 (fn [s] (pretty-item-view (lookup s items-dict))))
-               (/just 's)
-               (do (def selected (lookup s items-dict))
-                   (def item-number (match selected {:number 'n} n))
-                   (def pretty-item
-                     (add-comment-md
-                      (pretty-item-view selected)))
-                   (def edited-item (edit-in-editor! pretty-item))
-                   (def added-comment
-                     (string/unlines
-                      (take-while (fn [x] (not (eq? x comment-divider)))
-                                  (string/lines edited-item))))
-                   (put-str!
-                    (if (eq? added-comment "")
-                      "Item was not updated."
-                      (add-comment! machine item-number added-comment))))
-               _ (put-str! "No selection made."))
-        (do
-          (put-str! (pretty-table-header))
-          (map (fn [d] (put-str! (first d))) filtered-items)
-          (put-str! (summary (items-name item-flag) items)))))))
+      (match (lookup-maybe :fancy options)
+        [:just #t] (match (fzf-select-with-preview!
+                       (map first filtered-items)
```

Comment:
This feels kind of confusingly formatted. I'd say ideally in a `match` the pattern is to the lhs of the clause. Also, I think traditionally arguments are indented more than the function. 

(This will of course depend on your editor, but for me, setting the filetype to clojure makes the editor do quite sensible indentation)

Reply:
I tried to clean it up a bit in https://github.com/radicle-dev/radicle/pull/652/commits/fe520a861557b03e595ca63cc12e4fcb5ca71af5